LEGO1/act3state.cpp

```diff
@@ -0,0 +1,13 @@
+#include "act3state.h"
+
+// OFFSET: LEGO1 0x10072270 STUB
```

Comment:
At this address, there is `Act3::Act3`.
`Act3State` is created in ` LegoObjectFactory::Create` int 10009a90.